### PR TITLE
Refs [TASK] [36126] Create UI Hero Screen

### PR DESCRIPTION
**Pull request by thanhvinh2k**
## Related Tickets
- [#Task 36126: Create UI] (https://edu-redmine.sun-asterisk.vn/issues/36126)
## WHAT
- Create UI for hero screen
## Evidence (Screenshot or Video)
hero2screen
## Review Checklist

Category | View Point | Description | Self review | Reviewer2 (name)
--- | --- | --- | --- | ---
Conventions | Sun* coding conventions | https://github.com/framgia/coding-standards/tree/master/eng/android |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Conventions | Kotlin coding conventions | https://kotlinlang.org/docs/coding-conventions.html |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Sun* Redmine working process  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md |<li>- [ ] yes</li>|<li>- [ ] yes</li>  

## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/res/layout/fragment_champion.xml [hunk 1]
```diff
@@ -2,6 +2,43 @@
...
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/dp_10"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent">
+
+        <androidx.appcompat.widget.SearchView
+            android:id="@+id/editTextSearch"
+            android:layout_width="@dimen/dp_240"
```
Comment (datdt-1935) [inline app/src/main/res/layout/fragment_champion.xml]: không fix cứng width/height nhé

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -7,4 +7,5 @@
     <string name="notify_reselect_item">You have already selected</string>
     <string name="notify_exit_app">Exit app?</string>
     <string name="search">Search Hero</string>
+    <string name="search_champion_screen">Search</string>
```
Comment (tuanlvt-1117) [inline app/src/main/res/values/strings.xml]:
```suggestion
    <string name="search">Search</string>
```

File: app/src/main/res/values/colors.xml [hunk 3]
```diff
@@ -18,4 +18,6 @@
     <color name="fern">#57BA73</color>
     <color name="heliotrope">#9E6DF7</color>
     <color name="curious_blue">#319ED3</color>
+    <color name="white">#ffffff</color>
```
Comment (datdt-1935) [inline app/src/main/res/values/colors.xml]: màu này có sẵn rồi em `@android:color/white`

File: app/src/main/res/layout/item_layout_hero.xml [hunk 4]
```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_margin="@dimen/dp_15"
+    android:backgroundTint="@color/big_stone">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="@dimen/dp_170"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/item_layout_hero.xml:10]: dùng percent & ratio trong ConstraintLayout

Comment (thanhvinh2k) [reply app/src/main/res/layout/item_layout_hero.xml:10]: Dạ em có dùng percent và ratio để set lại cho cái MaterialCardview rồi mà anh

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**